static/sass/_snapcraft_details_heading.scss

```diff
@@ -31,33 +33,33 @@
     }
 
     &__publisher {
-      margin-top: $sp-x-small;
+      margin-top: -.5rem;
```

Comment:
Why changing to negative margin? How it is related to autoplay videos?

Reply:
Oh, we are changing layout of snap heading as well...

Reply:
Yeah @therealjuan wanted me to roll his PR changes (with fixes) into this